packages/react-instantsearch/src/connectors/connectAutoComplete.js

```diff
@@ -0,0 +1,61 @@
+import createConnector from '../core/createConnector';
+import {omit} from 'lodash';
+
+const getId = () => 'query';
```

Comment:
I think it's a notation used in every connector, sometime the id is not a single string, so we always have the same form to get the state id.